examples/src/retrievers/multi_vector_hypothetical.ts

```diff
@@ -89,9 +90,11 @@ const keyValuePairs: [string, Document][] = docs.map((originalDoc, i) => [
   originalDoc,
 ]);
 
-// The docstore to use to store the original chunks
-const docstore = new InMemoryStore();
-await docstore.mset(keyValuePairs);
+// The byteStore to use to store the original chunks
+const byteStore = new InMemoryStore<Uint8Array>();
+// Convert the byteStore to a docStore to use for retrieval
+const docStore = createDocumentStoreFromByteStore(byteStore);
+await docStore.mset(keyValuePairs);
```

Comment:
Examples should use `retriever.docstore.mset` or we could maybe add something like `retriever.addDocuments`

Reply:
I like the `addDocuments` method, will implement